test/test_helper.rb

```diff
@@ -80,6 +80,9 @@ def self.coerce(vrp)
 
     vrp[:vehicles].each{ |vehicle|
       vehicle.delete(:original_id)
+      next unless vehicle[:skills].to_a.empty? # nil or []
+
+      vehicle[:skills] = [[]]
```

Comment:
This doesn't correct the case if `vehicle[:skills]` is `["skill1", "skill2"]`.


We already have the correct fix in https://github.com/Mapotempo/optimizer-api/pull/274
you can cherry-pick the commit https://github.com/Mapotempo/optimizer-api/pull/274/commits/de47b5b00724d158bfa26be1452a98c39ebbe591 that way we wouldn't have a conflict.